### PR TITLE
[Issue 1567] Adds Cloudsoft links

### PR DESCRIPTION
**Pull request by laurenrother**
Links to Cloudsoft Riak integration tool were added in Community Projects in Advanced/Dev and on the Installation page of the Building/Ops page. The Building Ops page includes a description of the integration from the tool's README.

### CODE REVIEW

**Comment by laurenrother** [comment]
Fiiiiiiiinally!

File: source/languages/en/riak/ops/building/installing/index.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ detailed Riak installation instructions.
 
   * [[Installing Riak With Chef]]
 
+### Cloudsoft
+
+Cloudsoft provides tested, optimized Riak blueprints to help develop and deploy applications faster and easier on a wide range of environments. The AMP-Basho software will automatically deploy and manage Basho Riak and Riak Enterprise clusters using Apache Brooklyn's easy-to-use YAML blueprinting approach, combined with Cloudsoft's enterprise-supported Application Management Platform (AMP).
```
Comment (TheOnlyRew) [inline source/languages/en/riak/ops/building/installing/index.md]: There's been a lot of bikeshedding around this, but I at least have been trying to maintain an 80 character column width in the docs---especially in new content added to the docs. Could you go back and break this paragraph up?

**Comment by TheOnlyRew** [comment]
Looks good, save my concern regarding line length.

**Comment by TheOnlyRew** [comment]
+1

**Pull request merged**